pkg/builder/builder.go

```diff
@@ -111,9 +111,18 @@ func buildVolumeMounts(vdb *vapi.VerticaDB) []corev1.VolumeMount {
 	// container. Otherwise, you will get multiple mount points shared the same
 	// path, which will prevent any pods from starting.
 	if vdb.Spec.Local.DataPath != vdb.Spec.Local.DepotPath {
-		volMnts = append(volMnts, corev1.VolumeMount{
-			Name: vapi.LocalDataPVC, SubPath: vdb.GetPVSubPath("depot"), MountPath: vdb.Spec.Local.DepotPath,
-		})
+		if vdb.IsDepotVolumeEmptyDir() {
+			// If depotVolume is EmptyDir, the depot is stored in its own 'emptyDir' volume
+			vmts := []corev1.VolumeMount{
+				{Name: vapi.DepotMountName, MountPath: fmt.Sprintf("%s/%s", paths.LocalDataPath, vdb.GetPVSubPath("depot"))},
```

Comment:
I don't think we need this mount path. I think the /home/dbadmin/local-data should just point to things that are stored in the PV. The mount path the next line should be all that we need to setup the depot dir.